sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -1149,7 +1149,7 @@ def blocking_get(
       return self._lazy_iterator(state_key, coder)
     # Cache lookup
     cache_state_key = self._convert_to_cache_key(state_key)
-    cached_value = self._state_cache.get(cache_state_key, cache_token)
+    cached_value = self._state_cache.peek((cache_state_key, cache_token))
```

Comment:
I swapped the API to use a single key field since the internal implementation of the cache shouldn't care. Now the clients will pass in a tuple that represents the key.